Include the Anim user manual and tutorial (#257)
playcanvas/developer.playcanvas.com: content/en/user-manual/packs/components/anim.md

---
title: Anim
template: usermanual-page.tmpl.html
position: 1
---

> The anim component is used to connect an animstategraph asset and all of it’s required animation assets to a single entity.

Reviewer: ```suggestion
The anim component is used to connect an animstategraph asset and all of its required animation assets to a single entity.
```
